enigma/src/main/java/cuchaz/enigma/translation/mapping/serde/MappingFormat.java

```diff
@@ -24,7 +26,8 @@
 	TINY_V2(new TinyV2Writer("intermediary", "named"), new TinyV2Reader()),
 	TINY_FILE(TinyMappingsWriter.INSTANCE, TinyMappingsReader.INSTANCE),
 	SRG_FILE(SrgMappingsWriter.INSTANCE, null),
-	PROGUARD(null, ProguardMappingsReader.INSTANCE);
+	PROGUARD(null, ProguardMappingsReader.INSTANCE),
+	RGS(RGSWriter.INSTANCE, RGSReader.INSTANCE);
```

Comment:
Should better name the classes `RgsWriter` and `RgsReader`

Reply:
I don't see any real difference? I mean, I am fine with both, but RetroGuardSource is the name of the parser it is being used for so I think the capital abbreviations make more sense IMO. If you have any different reasoning, I'm fine with changing it.

Reply:
> If you have any different reasoning, I'm fine with changing it.

Code style :P
It's also e.g. CfrDecompiler and AsmObjectTranslator

Reply:
Okay, I'll change it, but do note the code I have now is fucking cursed.